node/coinstacks/solana/api/src/controller.ts

```diff
@@ -64,7 +65,48 @@ export class Solana implements BaseAPI, API {
   @Response<InternalServerError>(500, 'Internal Server Error')
   @Get('account/{pubkey}')
   async getAccount(@Path() pubkey: string): Promise<Account> {
-    return { pubkey } as Account
+    try {
+      let page = 1
+      let balance = '0'
+      let hasMore = false
+      let tokens: Array<TokenBalance> = []
+
+      do {
+        const { total, limit, items, nativeBalance } = (await heliusSdk.rpc.getAssetsByOwner({
+          ownerAddress: pubkey,
+          page: page++,
+          displayOptions: { showFungible: true, showNativeBalance: true, showZeroBalance: true },
+        })) as DAS.GetAssetResponseList & { nativeBalance: NativeBalance }
```

Comment:
god... :(

Reply:
yeah, they added the `displayOption` key and then didn't update the response payload... definitely a bit of amateur hour over there lol